src/main/java/com/minecolonies/core/colony/Colony.java

```diff
@@ -1349,6 +1348,10 @@ public ServerLevel getWorld()
     @Override
     public IRequestManager getRequestManager()
     {
+        if (requestManager == null)
```

Comment:
Also not sure how guaranteed this was.

Reply:
Like if it's accessed anyway midclass directly. Probably saver with the previous setup with method to set?

Reply:
the reason why we need to delay it is that during colony loading from nbt the manager had to be set later, which was done originally through a specific set call. The usages of the getter all seem to expect nonnull though, no one checks for it so should be fine :)